### DISCUSSION TITLE
Orca is Dead?

### DISCUSSION THREAD

**Discussion by midnightsmith89** [OrcaSlicer/OrcaSlicer#2518] [General]
Basically, the title. it seems that there has been no development in over two months and no new release. It seems bamboo studio has had at least seven releases since 1.7. trying to use orca slicer now gives you so many errors in the printer that it doesn't seem like it's safe to continue using

**Comment by darco415** [comment]
what are you talking about? lol...its been updated to 1.8 beta 2 days ago jeez

**Comment by Nukle0n** [comment]
There's no builds available yet, but also it doesn't give any errors on a Bambu at all for me so I don't know what @midnightsmith89 is doing. Just wait a couple days for public builds to show up or compile your own.

**Comment by CodyPrince** [comment]
I can see where he's coming from. Orca main page shows latest version sept 14 version 1.7. It is not very clear ho, or where to get "nightly" versions. Its also not very clear how or why to get the versions that are "Main" versions. Bambu has had a lot of versions out there, but they are really just bug fixes. if you count orca's nightly version, There's a new release every day which would be more like 60 releases to bambu's 7 releases. way more development on Orcas side. 

[go here](https://github.com/SoftFever/OrcaSlicer/actions) and see how many versions since 1.7 was released.
